src/commands/step-functions/README.md

```diff
@@ -0,0 +1,67 @@
+# step-functions commands
+
+You can use the `step-functions instrument` command to instrument your Step Functions with Datadog. This command enables instrumentation by subscribing step function logs to a [Datadog Forwarder](https://docs.datadoghq.com/logs/guide/forwarder/).
+
+You can also add the `step-functions instrument` command to your CI/CD pipelines to enable Datadog instrumentation for all of your step functions. Run the command after your normal serverless application deployment, so that changes made by this command do not get overridden by changes in the CI/CD pipeline.
```

Comment:
```suggestion
You can also add the `step-functions instrument` command to your CI/CD pipelines to enable Datadog instrumentation for all of your Step Functions. Run the command after your normal serverless application deployment, so that changes made by this command do not get overridden by changes in the CI/CD pipeline.
```